### ISSUE TITLE
TransmitterPDU is missing on dis7

### ISSUE THREAD

**Issue by semihkurt** [open-dis/open-dis-cpp#54]
TransmitterPdu is missing on include/dis7 though it is not deprecated in DIS  7 officially . In addition to this, it exists on Java repository  [OpenDIS Java](https://github.com/open-dis/open-dis-java/tree/master/src/main/java/edu/nps/moves/dis7). 

I am implementing DIS by using this repository thanks to you, I realized that something is missing in Radio Transmission part because newly added PDUs does not completely represent Radio Transmission. Then I found DIS6 vs DIS7 PDF and it seemsTransmitterPdu has not been deprecated on DIS 7.  

I also realized that if any Developer wants to use DIS 7:
1) They have to update classes inside Utils folder so I suggest that Utils folder should be created inside dis7 and dis6 folders.
2) They have to add missing PDUs on DIS 6 to DIS 7 project and update them.
3) They need to update TransmitterPDU for DIS 7 use  (ex: RadioEntityType becomes RadioType on dis7 and some other details.)

If you can grant me access on some branch, I can work on that branch and make a pull request to you.

**Comment by leif81** [comment]
@semihkurt thank-you for this report.

What you'll want to do is fork this repository into your user space, make the change in a branch there, and then submit a pull-request of that.

More info on that here:

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests

**Comment by semihkurt** [comment]
@leif81 Hello,
I've started to update TransmitterPdu in my fork and will put a pull request, thanks.

I found a document that compares dis 6 and dis 7 changes.  Could you validate this document whether it is correct [DIS v6 v7](https://www.sisostds.org/DesktopModules/Bring2mind/DMX/API/Entries/Download?Command=Core_Download&EntryId=36292&PortalId=0&TabId=105) because there are some additions required.

**Comment by leif81** [comment]
Yes that document is a good source for what's different between v6 and v7.

**Comment by semihkurt** [comment]
Thanks for validating the document Leif. I've made some initial changes on TransmitterPDU and made a pull request. If you allow me, I can contribute more as I currently work on DIS domain. I'm gonna add more missing PDUs into my fork and create new pull requests on further days. 

By the way, when should I close this thread ? After merge or something?

**Comment by leif81** [comment]
Sounds great. Yes let's leave open until merge .

**Comment by semihkurt** [comment]
Hey Leif,

In order for you to review two pull requests seperately, I've closed my Transmitter PDU pull request, after I finish it completely I am gonna open a new pull request from seperate branch.
